### DISCUSSION TITLE
Various styling issues

### DISCUSSION THREAD

**Discussion by arodriguezcortes** [twilio-labs/paste#1845] [Office hours] [answered]
Hi Paste team.

First I want to give you kudos, the components look great! I have some inquiries about styling I want to discuss with you.

1. I'm trying to create a tooltip inside a table cell similar to what you showcase in the documentation [here](https://paste.twilio.design/components/tooltip#focusable-element). However, I haven't been able to make the icon stay in the same line as the text and vertically aligned. You can find what I get in the images below. Can you give me a hint on how to make this work.

Using `display="block"` in the icon.
<img width="194" alt="Screen Shot 2021-09-07 at 2 43 01 PM" src="https://user-images.githubusercontent.com/88405338/132537240-ca1ff472-c78b-4c0f-a402-85e2b85450bc.png">

Using `display="inline-block"` in the icon.
<img width="186" alt="Screen Shot 2021-09-08 at 10 39 51 AM" src="https://user-images.githubusercontent.com/88405338/132540965-916b8c62-9c71-4061-b8dc-47ec4e78b310.png">

Using `display="inline"` in the icon.
<img width="412" alt="Screen Shot 2021-09-08 at 10 42 17 AM" src="https://user-images.githubusercontent.com/88405338/132541255-0c4b48be-0089-4757-84bd-a19304a262ee.png">

2. I want to display an alert box but seems that it doesn't have a `margin-bottom` by default and it's not in its props. I'm new to Paste, and I was considering using a `<Stack />` element to achieve the spacing but just want to confirm if you would also recommend that or have any suggestions regarding this.

<img width="802" alt="Screen Shot 2021-09-07 at 2 33 09 PM" src="https://user-images.githubusercontent.com/88405338/132541665-5b1ab43a-f84e-42d2-a088-7fe4ddc1b086.png">

Many thanks!

**Comment by arodriguezcortes** [comment, marked as answer]
I've solved the first question. I just realized you are using a `<Box />` with `display: flex` in the example 😬

I'd appreciate any recommendation regarding the second question though.

Thanks!

**Reply by nkrantz** [reply]
Hey @arodriguezcortes! Welcome to Paste 😄

You're correct that Alert doesn't have a `marginBottom` prop. That's because we treat our components as the building blocks of the UI, and we suggest using Layout components to format your spacing and to compose with multiple components. Stack is definitely an option, however I'd probably recommend using a [Box](https://paste.twilio.design/primitives/box) with a `marginBottom`. That way, no matter where your Alert is placed (within a Stack or outside of a Stack), it will retain its spacing.

Not sure what you're using for the paragraph below the Alert, but if you're using the [Text Primitive](https://paste.twilio.design/primitives/text), I'll note that that accepts a `marginTop`, which could be another option for you.

**Reply by arodriguezcortes** [reply]
Hi @nkrantz. Thanks a lot for your response, it's very helpful.
